hudi-client/hudi-client-common/pom.xml

```diff
@@ -275,6 +300,45 @@
           <skip>false</skip>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>io.fabric8</groupId>
+        <artifactId>docker-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>prepare-it-database</id>
+            <phase>pre-integration-test</phase>
+            <goals>
+              <goal>start</goal>
+            </goals>
+            <configuration>
+              <images>
+                <image>
+                  <name>amazon/dynamodb-local:${dynamodb-local.version}</name>
```

Comment:
It's because ```ITTestDynamoDBBasedLockProvider``` is located under ```hudi-client```, ```TestZookeeperBasedLockProvider``` is also located under ```hudi-client```. Do you want me to relocate it to ```hudi-integ-test```?